providers/capv/on-capz/test_data/helloworld_values.yaml

```diff
@@ -0,0 +1,15 @@
+clusterName: {{ .ClusterName }}
+organization: {{ .Organization }}
+ingress:
+  annotations:
+    kubernetes.io/tls-acme: "true"
+    cert-manager.io/cluster-issuer: letsencrypt-giantswarm
+  hosts:
+    - host: hello-world.{{ .ClusterName }}.gaws.gigantic.io
```

Comment:
Wrong domain. Should this file be here, or is it yet out of scope because of `ExternalDnsSupported: false`?

Reply:
Yeah. You are right. I will remove this file.